webpack/assets/javascripts/react_app/pages/AuditsPage/AuditsPage.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon } from 'patternfly-react';
+
+import PageLayout from '../common/PageLayout/PageLayout';
+import AuditsList from '../../components/AuditsList';
+import Pagination from '../../components/Pagination/Pagination';
+
+const toolbarBtn = (
+  <Button
+    href="http://www.theforeman.org/manuals/1.21/index.html#4.1.4Auditing"
```

Comment:
Please do not hardcode versions. Also use https URLs

Reply:
now receiving the href from the server 👍